livox_ros_driver/launch/livox_lidar.launch

```diff
@@ -10,7 +10,8 @@
 	<arg name="rviz_enable" default="false"/>
 	<arg name="rosbag_enable" default="false"/>
 	<arg name="cmdline_arg" default="$(arg bd_list)"/>
-	<arg name="msg_frame_id" default="livox_frame"/>
+	<arg name="lidar_msg_frame_id" default="livox_frame"/>
```

Comment:
Maybe I should rename the simulation package because it's easier, but the TF tree of Livox is the following:

livox_base > livox > livox_imu_link

Where livox_base is the mounting link, livox is the reference of the Lidar data and livox_imu_link is the reference for IMU data.

Maybe I could rename livox to livox_frame in simulation, but here you should rename  the IMU link to livox_imu_link or call it livox_imu_frame to differentiate the data coming from a different position. WDYT?

Reply:
I think that these are just non-breaking defaults. The idea would be to explicitly set these frame IDs where we include this launch file.